lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class PrometheusMetrics {
+  constructor (circuit) {
+    this.circuit = circuit;
+    // this.client = require('prom-client');
+    this.collect();
+  }
+
+  collect (prefix) {
+    this.collectDefaultMetrics = this.client.collectDefaultMetrics({ prefix: prefix, timeout: 5000 });
+    this.circuit.on('open', () => {
```

Comment:
I think this code should probably all be in the constructor, but in any case, we should be collecting metrics for circuit events, so I don't think `clear()` makes sense here (and below). Instead, we should be counting these events with a [counter](https://www.npmjs.com/package/prom-client#counter).

```suggestion
    this.circuit.on('open', () => {
    this.counters.forEach( counter => {
      counter.inc();
    });
```